graphbolt/src/concurrent_id_hash_map.cc

```diff
@@ -83,12 +83,22 @@ torch::Tensor ConcurrentIdHashMap<IdType>::Init(
   const int64_t num_threads = torch::get_num_threads();
   std::vector<size_t> block_offset(num_threads + 1, 0);
   // Insert all elements in this loop.
+  torch::parallel_for(
+      num_seeds, num_ids, kGrainSize, [&](int64_t s, int64_t e) {
+        for (int64_t i = s; i < e; i++) {
+          InsertAndSetSmaller(ids_data[i], static_cast<IdType>(i));
+        }
+      });
```

Comment:
Can't we just insert all elements in the previous loop? Because we use the SetSmaller variant, the first num_seeds will be guaranteed to be inserted anyway.